hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/mapreduce/SyncTable.java

```diff
@@ -546,6 +567,19 @@ private boolean syncRowCells(Context context, byte[] rowKey, CellScanner sourceC
               if (put == null) {
                 put = new Put(rowKey);
               }
+              if (ignoreTimestamp) {
```

Comment:
Seems to duplicate the if-branch above. Could be moved into a separate method.

Reply:
Indeed, fixed on latest commit.